src/__tests__/plugin.js

```diff
@@ -109,4 +109,30 @@ describe('gatsby-remark-embedder', () => {
 
     expect(remark.stringify(processedAST)).toMatchSnapshot();
   });
+
+  it('can execute custom transformers', async () => {
+    const markdownAST = getMarkdownASTForFile('CustomTransformer');
+
+    const transformer = {
+      shouldTransform: jest.fn(url => url.startsWith('https://some-site.com')),
+      getHTML: jest.fn(url => `<iframe href="${url}"></iframe>`),
+    };
+
+    const processedAST = await plugin(
+      { cache, markdownAST },
+      { customTransformers: [transformer] }
+    );
+
+    expect(transformer.shouldTransform).toHaveBeenCalledTimes(2);
+    expect(transformer.shouldTransform).toHaveBeenCalledWith(
+      'https://some-site.com/id/abc'
+    );
+    expect(transformer.shouldTransform).toHaveBeenCalledWith(
+      'https://some-other-site.com/'
+    );
+
+    expect(transformer.getHTML).toHaveBeenCalledTimes(1);
+
+    expect(remark.stringify(processedAST)).toMatchSnapshot();
```

Comment:
I'd switch this to `toMatchInlineSnapshot()`

Reply:
Thanks for the feedback! I have just updated the test.

Cheers!

Reply:
All other tests use `toMatchSnapshot`, so why do different here? 🤔

Reply:
@kentcdodds All other tests use `toMatchSnapshot`, so why do different here? 🤔

Reply:
I'd suggest it for all of them

Reply:
Why do you prefer inline over separate files?

This will make the test file a lot longer, no? 🤔

Reply:
Especially for the kitchensink

Reply:
@kentcdodds Why do you prefer inline over separate files?

This will make the test file a lot longer (especially for the kitchensink test), no? 🤔

Reply:
I haven't looked super closely, but I like to avoid large snapshot files when I can: https://kcd.im/snapshots

Feel free to ignore my advice. It's really not a huge deal.

Reply:
@cms I think leaving the snapshot file for this PR is fine, so please revert back to `toMatchSnapshot` for consistency.

@kentcdodds I was already thinking about refactoring all tests, since each case looks extremely the same and _is_ kinda the same, just for another built-in transformer.
Will take that article into account when doing so 🙂